spec/jobs/plexer_job_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+describe PlexerJob, type: :job do
+  let(:druid) { 'bj102hs9687' }
+  let(:version) { 1 }
+
+  it 'descends from ApplicationJob' do
+    expect(described_class.new).to be_an(ApplicationJob)
+  end
+
+  # future: does some replication logic
```

Comment:
`TODO` just to make it more conventional/searchable?

Reply:
sure

Reply:
I didn't conceive of there being a large gap of time between this PR and the uptake of future work such that it would be easy to "miss" that the pieces are still skeletal.